### DISCUSSION TITLE
Can't change git sync interval

### DISCUSSION THREAD

**Discussion by seonwoolee** [requarks/wiki#3550] [Error / Bug Report]
**Describe the bug**
I'm trying to change the git sync interval from 5 minutes to 1 hour, but it does not change. I can hit apply, and it will say that the sync interval has changed to 1 hour, but when I refresh the page it goes back to 5 minutes. 

**To Reproduce**
1. Go to administration, storage
2. Select git, scroll down to sync schedule
3. Change from 5 minutes to 1 hr - minutes 
4. Click apply

**Expected behavior**
The sync interval changes to 1 hr. 

**Screenshots**

**Host Info (please complete the following information):**
 - OS: Docker on a ZFS backed Arch Linux host
 - Wiki.js version: 2.5.136
 - Database engine: Postgres 12.4

**Additional context**
I've checked the logs for the Docker container and there's no warnings whatsoever, just a bunch of info lines saying git pushes and pulls have been executed.

**Comment by eeriksp** [comment]
Same here using Ubuntu 20 without Docker and an Sqlite database.

**Comment by sapphire008** [comment]
Same issue. Wiki.js Version 2.5.170. MySQL 10 database. Cloudlinux OS.

**Comment by adrian-martinez-interactiv4** [comment]
Same here with Wiki.js 2.5.170
